2024-04-24 16:16:08,531 INFO    MainThread:13836 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-24 16:16:08,531 INFO    MainThread:13836 [wandb_setup.py:_flush():76] Configure stats pid to 13836
2024-04-24 16:16:08,532 INFO    MainThread:13836 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Hilly Segal\.config\wandb\settings
2024-04-24 16:16:08,532 INFO    MainThread:13836 [wandb_setup.py:_flush():76] Loading settings from C:\HumanChoicePredictionCloned\wandb\settings
2024-04-24 16:16:08,532 INFO    MainThread:13836 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-24 16:16:08,532 INFO    MainThread:13836 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-24 16:16:08,533 INFO    MainThread:13836 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'StrategyTransfer.py', 'program_abspath': 'C:\\HumanChoicePredictionCloned\\StrategyTransfer.py', 'program': 'C:/HumanChoicePredictionCloned/StrategyTransfer.py'}
2024-04-24 16:16:08,533 INFO    MainThread:13836 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-24 16:16:08,534 INFO    MainThread:13836 [wandb_init.py:_log_setup():521] Logging user logs to C:\HumanChoicePredictionCloned\wandb\run-20240424_161608-ak1ywsda\logs\debug.log
2024-04-24 16:16:08,535 INFO    MainThread:13836 [wandb_init.py:_log_setup():522] Logging internal logs to C:\HumanChoicePredictionCloned\wandb\run-20240424_161608-ak1ywsda\logs\debug-internal.log
2024-04-24 16:16:08,535 INFO    MainThread:13836 [wandb_init.py:init():561] calling init triggers
2024-04-24 16:16:08,535 INFO    MainThread:13836 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-24 16:16:08,536 INFO    MainThread:13836 [wandb_init.py:init():611] starting backend
2024-04-24 16:16:08,536 INFO    MainThread:13836 [wandb_init.py:init():615] setting up manager
2024-04-24 16:16:08,547 INFO    MainThread:13836 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-04-24 16:16:08,556 INFO    MainThread:13836 [wandb_init.py:init():623] backend started and connected
2024-04-24 16:16:08,568 INFO    MainThread:13836 [wandb_init.py:init():715] updated telemetry
2024-04-24 16:16:08,638 INFO    MainThread:13836 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-24 16:16:09,651 WARNING MainThread:13836 [wandb_init.py:init():1203] interrupted
Traceback (most recent call last):
  File "C:\ProgramData\anaconda3\envs\final_project_env\lib\site-packages\wandb\sdk\wandb_init.py", line 1181, in init
    run = wi.init()
  File "C:\ProgramData\anaconda3\envs\final_project_env\lib\site-packages\wandb\sdk\wandb_init.py", line 751, in init
    result = run_init_handle.wait(
  File "C:\ProgramData\anaconda3\envs\final_project_env\lib\site-packages\wandb\sdk\lib\mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "C:\ProgramData\anaconda3\envs\final_project_env\lib\site-packages\wandb\sdk\lib\mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "C:\ProgramData\anaconda3\envs\final_project_env\lib\site-packages\wandb\sdk\lib\mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "C:\ProgramData\anaconda3\envs\final_project_env\lib\threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "C:\ProgramData\anaconda3\envs\final_project_env\lib\threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-04-24 16:16:14,946 WARNING MsgRouterThr:13836 [router.py:message_loop():77] message_loop has been closed
